Source/inv.cpp

```diff
@@ -2065,7 +2065,11 @@ BOOL UseStaff()
 {
 	if (pcurs == CURSOR_HAND) {
 		if (plr[myplr].InvBody[INVLOC_HAND_LEFT]._itype != ITYPE_NONE
-		    && plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_STAFF
+#ifdef HELLFIRE
+		    && (plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_STAFF || plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_UNIQUE)
+#else
+		   && plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_STAFF

```

Comment:
Looks like I missed a space, if you give me permission for you fork then I can do some of thease simple corrections my self :)
```suggestion
		    && plr[myplr].InvBody[INVLOC_HAND_LEFT]._iMiscId == IMISC_STAFF
```